client/gefyra/api/install.py

```diff
@@ -51,7 +51,10 @@ def install(
     from gefyra.configuration import ClientConfiguration
 
     config = ClientConfiguration(
-        kube_config_file=kubeconfig, kube_context=kubecontext, ignore_docker=True
+        kube_config_file=kubeconfig,
+        kube_context=kubecontext,
+        ignore_docker=True,
+        registry_url=kwargs.get("registry_url", ""),
```

Comment:
Why not make a dedicated kw? I think that would be cleaner and nicer for code completion in IDEs

Reply:
Because of the construction of GefyraInstallOptions in lines 66-67. Making regstry_url a dedicated keyword pops it from kwargs leading to the option not being respected anymore

Reply:
Still I think it would be better to make it explicit, just for code completion and typing.
You could add it to the `kwargs` in case a preset is set.